### ISSUE TITLE
"Open Devtools Pane" does not work in browser-based Codespaces

### ISSUE THREAD

**Issue by filandra** [microsoft/vscode-livepreview#513]
**Issue replication:** The issue can be replicated by running Codespaces in the browser and pressing the "Open Devtools Pane" button. I was able to replicate this issue on all the browsers that I tried, namely:

- On Ubuntu 20.04: Firefox 114.0, Chrome 114.0.5735.106
- On iOS 16.5: Safari 16.4.1
- On Meta Quest 2 54.0: Meta Quest Browser 27.3.0

**The issue:** After pressing the button, nothing happens - not even an error message is displayed.

_Option 1)_ Browser-based Codespaces have inherent limitation and they can't display the Devtools Pane. In this case either the button should be disabled or a message should be displayed saying that this feature is unsupported.

_Option 2)_ Browser-based Codespaces are supposed to be able to open the Devtools Pane and this should be fixed.

Personally I hope it's the second option as I was really looking forward to using browser-based Codespaces for web development on a mobile device. It's really hard to work without the Devtools Pane due to the fact, that you can't easily access the Developer Tools on mobile browsers.

In the meantime is there some other extension that could replicate the functionality of Devtools Pane and works in the browser-based Codespaces?

**Comment by andreamah** [comment]
Although this might not be perfect (since it includes any browser errors), you can use `ctrl`+`shift`+`i` on most browsers to access the devtools pane. But I see that the dropdown menu action doesn't work.

**Comment by filandra** [comment]
Yes, the `ctrl`+`shift`+`i` works on most desktop browsers, but not on mobile browsers. For example when developing on an Android tablet or an iPad accessing the devtools pane is not trivial. That's why I wanted to use Live Preview in the first place, because I hoped it would solve this issue.

For others who might run into the same issue - the tool [eruda](https://github.com/liriliri/eruda) offers some sort of solution for this problem, but it's not ideal.
